mix/qml/DebugBasicInfo.qml

```diff
@@ -0,0 +1,109 @@
+import QtQuick 2.2
+import QtQuick.Controls 1.1
+import QtQuick.Layouts 1.0
+import QtQuick.Controls.Styles 1.1
+
+ColumnLayout {
+	property string currentStep
+	property string mem
+	property string stepCost
+	property string gasSpent
+	spacing: 0
+	RowLayout {
```

Comment:
Repeating elemets should be defined as components to avoid copy-pasting blocks of similair code